frontend/src/views/HomeView.vue

```diff
@@ -1,10 +1,96 @@
 <template>
   <div class="home">
+    <h2>アサインされているタスク</h2>
+    <table>
+      <thead >
+        <tr>
+          <th>チーム</th>
+          <th>タスクID</th>
+          <th>タイトル</th>
+          <th>担当者</th>
+          <th>作成日時</th>
+          <th>操作</th>
+        </tr>
+      </thead>
+      <tbody >
+        <tr v-for="mytask in mytasks" :key="mytask.id">
+          <td>{{mytask.team_id}}</td>
+          <td>
+            {{mytask.id}}
+          </td>
+          <td>
+            {{mytask.title}}
+          </td>
+          <td>
+            {{mytask.assignee_id}}
+          </td>
+          <td>
+            {{mytask.created_at}}
+          </td>
+        </tr>
+    </tbody>
+    </table>
+    <h2>所属しているチーム</h2>
+    <table>
+      <thead >
+        <tr>
+          <th>チームID</th>
+          <th>チーム名</th>
+        </tr>
+      </thead>
+      <tbody >
+        <tr v-for="(myteam, id) in myteams" :key="id">
+          <td>  {{myteam.id}}</td>
+          <td>
+            {{myteam.name}}
+          </td>
+        </tr>
+      </tbody>
+    </table>
   </div>
 </template>
 
 <script>
+import axios from 'axios';
+import { onMounted, ref } from 'vue';
+
 export default {
   name: 'HomeView',
+  setup(){
+    const mytasks = ref([])
+    const myteams = ref([])
+
+    const fetchMyTasks = async() => {
+      const url = "http://localhost:8080/api/me/tasks"
+      const res = await axios.get(url)
+      mytasks.value = res.data
+    }
+
+    const fetchMyTeams = async() => {
+      const url = "http://localhost:8080/api/me/teams"
+      const res = await axios.get(url)
+      myteams.value =res.data
+    }
+
+    onMounted(async () => {
+        Promise.all([fetchMyTasks(), fetchMyTeams()]);
```

Comment:
内容としてはほぼOKです。

```suggestion
     await Promise.all([fetchMyTasks(), fetchMyTeams()]);
```

の方がいいかな。 async もつけているので、await すると良さそう。周辺実装にもよりますが、Promiseを返す関数をそのままにしておくと、エラーが握りつぶされてしまうので、 await つけて throw されるようにしておく方が良いことが多いです。

後で講義で確認していきましょう。

Reply:
@mori-kei 講義中でもやりましたが、Vue.js の環境だとハンドルされない例外も表示を出してくれる機能があったので、エラーが起きたことは表示されますね（ユーザーへのフィードバックはされないのでそれは自分でやる必要があります）。